src/AsyncEventSource.h

```diff
@@ -113,14 +116,17 @@ class AsyncEventSource: public AsyncWebHandler {
     String _url;
     LinkedList<AsyncEventSourceClient *> _clients;
     ArEventHandlerFunction _connectcb;
+    ArEventHandlerFunction2 _disconnectcb;
   public:
     AsyncEventSource(const String& url);
     ~AsyncEventSource();
 
     const char * url() const { return _url.c_str(); }
     void close();
     void onConnect(ArEventHandlerFunction cb);
+    void onDisconnect(ArEventHandlerFunction2 cb);
     void send(const char *message, const char *event=NULL, uint32_t id=0, uint32_t reconnect=0);
+    bool try_send(const char *message, const char *event=NULL, uint32_t id=0, uint32_t reconnect=0);
```

Comment:
why overloading ? in both cases, there is no send guarantee. 
I would just return false bool on the original method if the idea is to get a feedback whether the message was enqueued or not